libs/components/theme/src/lib/styles/themes/modern/_forms.scss

```diff
@@ -2,7 +2,7 @@
 
 .sky-theme-modern {
   .sky-control-help-container:not(:empty) {
-    margin-left: var(--sky-margin-inline-sm);
+    margin-left: var(--sky-margin-inline-xs);
```

Comment:
This value is already set in the default styles, so this line can be removed entirely. See: https://github.com/blackbaud/skyux/blob/main/libs/components/theme/src/lib/styles/_forms.scss#L63-L65